floris/optimization/layout_optimization/layout_optimization_pyoptsparse.py

```diff
@@ -19,9 +19,35 @@ def __init__(
         storeHistory='hist.hist',
         hotStart=None,
         enable_geometric_yaw=False,
+        use_value=False,
     ):
-        super().__init__(fmodel, boundaries, min_dist=min_dist,
-                         enable_geometric_yaw=enable_geometric_yaw)
+        """
+        _summary_ TODO: write summary and describe the remaining arguments
```

Comment:
@ejsimley if you want I'm happy to take a crack at this and commit to your branch.  Also, elsewhere in FLORIS we have the connection that the class docstring comes before the init function, in the same commit I can move this above?

Reply:
There are a few arguments that aren't described in the docstring yet (timeLimit, storeHistory, hotStart). @bayc, it's not a big issue, but since we're updating this now, would you mind adding the descriptions? Or if you just want to write them here I can update the code.